logfire/_internal/formatter.py

```diff
@@ -386,6 +386,12 @@ def get_node_source_text(node: ast.AST, ex_source: executing.Source):
     This happens sometimes due to Python bugs (especially for older Python versions)
     in the source positions of AST nodes inside f-strings.
     """
+    # Check for await expressions first, before attempting to unparse
+    if isinstance(node, ast.Await):
```

Comment:
There could also be an await deeper within the node

Reply:
these tests should just be skipped in 3.8. but also don't parse the source, you already have an ast node.